okta/help.md

```diff
@@ -568,6 +611,7 @@ by Okta themselves, or constructed by the plugin based on the information it has
 
 # Version History
 
+* 3.3.0 - New actions for getting factors and sending a push verify
```

Comment:
Minor nit, follow changelog style for new actions: https://komand.github.io/python/style.html#changelog

Reply:
NOT MINOR, I NEED TO CONFORM